pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -106,12 +108,14 @@ var _ = Describe("Pod Network", func() {
 			Mtu:     1410,
 			Gateway: gw}
 
+		ipv4CidrStr = api.DefaultVMCIDR
```

Comment:
Please explain why this extra variable is need.

Reply:
I am trying to concentrate all IP addresses configuration in one place and the CIDR is related.
I these extra variables in case there would be a change to where the CIDR's are taken from,
It makes it easier to test and debug this way.

Reply:
You can always substitute this new variable with `api.DefaultVMCIDR` directly, I see no reason to have another naming in between.

If in the future there will be a good enough reason for it, it can be added then.

Reply:
from another conversation:
In my opinion its easier to understand and to maintain these kind of tests by keeping all configurations in one place, in this case the configuration is the IP addresses.
The fact that we take the default CIDRs from api shouldn't matter, if there would be any change and the default CIDR will not be in the api anymore (which I think it shouldn't be there but that for another day (; ) , we would change the variable that represent it w/o touching the test itself cussing potential bugs.
This is also my main reason to switch from hard-coded addresses to representative variables.

Reply:
I do not really understand your reasoning here. If `DefaultVMCIDR` changes location you still need to change this module.
Also, it is a good thing for this test to fail if that `DefaultVMCIDR` changes in any way... The test **is** dependent on it and changes to it is reflected through the test.

Reply:
Agreed to disagree, I will remove those variables for the sake of this PR to merge before code freeze
I appropriate the feedback, thanks :)